pkg/build/api/types.go

```diff
@@ -82,6 +82,9 @@ type BuildStatus struct {
 	// Duration contains time.Duration object describing build time.
 	Duration time.Duration
 
+	// Pushspec contains a resolved Docker pushspec computed from Build.Spec.Output.To.
+	Pushspec string
```

Comment:
We aren't supporting multiple today, so let's not add in things we're not quite clear on.